docs/resources/encapsulation_context.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Typo in "Grandchild contex"

Reply:
https://github.com/fastify/fastify/pull/2725